modules/http/src/main/java/org/mule/module/http/internal/listener/grizzly/GrizzlyServerManager.java

```diff
@@ -233,6 +242,21 @@ public void dispose()
         }
     }
 
+    private IdleTimeoutFilter createTimeoutFilter(boolean usePersistentConnections, int connectionIdleTimeout)
+    {
+        int timeout;
+        if (usePersistentConnections && serverTimeout < connectionIdleTimeout)
+        {
+            // Avoid interfering with the keep alive timeout but secure plain TCP connections
+            timeout = connectionIdleTimeout * 2;
```

Comment:
I understand the reason for doing this, but why `* 2` instead of `+ 100`, for instance?

Reply:
Just a random delay. I think 500 ms should be enough, 100 seems to narrow.